ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -52,6 +70,8 @@ public class ThreeDSecureRequest implements Parcelable {
     private Boolean cardAddChallengeRequested;
     private ThreeDSecureV2UiCustomization v2UiCustomization;
     private ThreeDSecureV1UiCustomization v1UiCustomization;
+    private @ThreeDSecureUiType int uiType;
+    private List<Integer> renderType;
```

Comment:
```suggestion
    private List<Integer> renderTypes;
```
🧶 : It'd be good to align the name of the backing property to the getter / setter combo.

Reply:
Ah good call - one day I'll get used to Android's properties all over and different methods for getters / setters 🙈 . Updated here: https://github.com/braintree/braintree_android/pull/738/commits/08f63fd4264e5926a359a9a9e9ec55bf16625a3c